README.md

```diff
@@ -1,23 +1,28 @@
 # caus-python
```

Comment:
General comment: you seem to have used a formatter. It would be good to specify in the README which formatter was used and if applicable the config for the formatter

Reply:
Nope, I didn't.

Reply:
ok, then you did the formatting manually?
then maybe introducing a formatter would be an option
I personally like https://github.com/psf/black, but again, if you have other preferences, choose as you like